packages/dev/loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -2229,7 +2224,24 @@ export class GLTFLoader implements IGLTFLoader {
     /**
      * @internal
      */
-    public _loadMaterialAsync(
+    private async _ensurePbrMaterialClassesAsync(): Promise<void> {
+        if (!this._pbrMaterialClass || !this._pbrMaterialAdapterClass) {
+            const openpbrExt = this._extensions.find((extension) => extension.name === "KHR_materials_openpbr");

```

Comment:
Won't this always be true since it is just checking if we have a loader extension for `KHR_materials_openpbr` (which we always would)? Should this instead be:
```suggestion
            const openpbrExt = this.isExtensionUsed("KHR_materials_openpbr");
```